### DISCUSSION TITLE
Workaround or mirror for broken firmware host "source.sierrawireless.com" ?

### DISCUSSION THREAD

**Discussion by noitauqe** [danielewood/sierra-wireless-modems#74] [Q&A] [answered]
Links to firmware in this guide are broken.  The hosting server "source.sierrawireless.com" reports the following error in response to every request.

> 
> Server Error in '/' Application.
> Runtime Error
> Description: An exception occurred while processing your request. Additionally, another exception occurred while executing the custom error page for the first exception. The request has been terminated.
> 

Likewise in my original search for MMS support, a vital document is indexed by search engines but also no longer accessible:
https://source.sierrawireless.com/resources/#tags=FXT009%7CApplication+Notes+%26+Code+Samples

Unfortunately the internet Wayback Machine (archive.org) seems to lack snapshots of these resources.

**Comment by ghost** [comment]
Any way to find a mirror? Wayback not showing anything either.

**Reply by nilgrant** [reply]
Sierra Wireless was hit with a ransomware attack 2 weeks ago, and their firmware site has been down ever since. I have the same problem so I reached out to their support and have not received any response. I found an old version of the firmware (2016) on techship.com, and was able to modify the script to flash it successfully, but it appears this firmware is too old and it doesn't work.

**Reply by tmuehlhoff** [reply]
they're back up.

**Comment by danielewood** [comment, marked as answer]
I still had a cached copy from some updates I did in December.

I have temporarily hosted the firmware on my own server, I will be removing it once Sierra Wireless is back online as I dont want them to DMCA me.

See the latest release.
https://github.com/danielewood/sierra-wireless-modems/releases/tag/v0.0.2

**Reply by noitauqe** [reply]
Thank you for fixing so quickly.

Do you happen to have a copy of to the MMS application note OP FXT009 search engine indexed as below?  [Are you aware of MMS functionality and how to access it for any of the lower rung devices?]

> EM7305; EM7330; EM7355; EM7411; EM7421; EM7430; EM7431; EM7455; EM7511 ... AirPrime - Multimedia Message Service (MMS) - Application note.

**Reply by danielewood** [reply]
I do not. MMS was never a consideration for my uses.

**Reply by nilgrant** [reply]
Daniel, thanks for the update. Worked perfectly!
